test-requirements.txt

```diff
@@ -8,3 +8,4 @@ httpx  # For FastAPI tests
 
 odoo-addon-shopinvader_api_customer @ git+https://github.com/shopinvader/odoo-shopinvader@refs/pull/1551/head#subdirectory=setup/shopinvader_api_customer
 odoo-addon-shopinvader_api_settings @ git+https://github.com/shopinvader/odoo-shopinvader@refs/pull/1555/head#subdirectory=setup/shopinvader_api_settings
+odoo-addon-product_get_price_helper @ git+https://github.com/OCA/product-attribute@refs/pull/1954/head#subdirectory=setup/product_get_price_helper
```

Comment:
@thienvh332 Can you remove this test dependency since the related PR is merged?

Reply:
@thienvh332 I made the change in https://github.com/shopinvader/odoo-shopinvader/pull/1587 and I'll merge it. Thank you for the fix and the enhancement!